domains/crystalperson.is-app.tech.json

```diff
@@ -0,0 +1,15 @@
+{
+    "domain": "is-app.tech",
+    "subdomain": "crystalperson",
+
+    "owner": {
+	"username": "jinix6",
+        "email": "lipanghadai@gmail.com"
+    },
+
+    "records": {
+        "TXT": ["62a3d6a2dfcc810b777f1838ff3f67"]
```

Comment:
The format of the txt record will be like this
`"TXT": [ { "name": "@", "value": "example_verification=1234567890" } ]`

Reply:
@jinix6 ^^